IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1078,7 +1078,10 @@ body {
 <script type="text/javascript">
 game_options = {
   use_query_story: false,
-  set_page_title: true
+  set_page_title: true,
+  recording_url: '../../../play/$entry_id/transcribe',
```

Comment:
File under "probably needed to be that way in 2014 for reasons now forgotten". Seems to work with absolute paths now, so I changed it per suggestion.